### DISCUSSION TITLE
Future path

### DISCUSSION THREAD

**Discussion by Bilza95** [xerpi/vita2hos#12] [Q&A]
Is the plan to emulate full PS Vita incl. Games?
Are you planning a gui with .elf selector?
Edited: removed question due to questionable legality

**Comment by TSRBerry** [comment]
> Would this work if we used tinfoil to install the .nsp as an app 

That's piracy and won't be supported.

**Reply by Bilza95** [reply]
Sorry I mean installing the .nsp for vita2hos - surely homebrew installations on switch is fine?

**Comment by ghost** [comment]
Nope—any .nsp files not made with the Nintendo SDK are considered piracy.

**Reply by Bilza95** [reply]
Oh wow - didn't know that. That's news to me and fair enough then - disregard that part of the post
